blueprints/redirect-upload-requests/blueprint.json

```
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "Redirect upload requests",
```

This title could be clearer:

```suggestion
		"title": "Serve media files from another host",
```

---

I went ahead and updated the title. I'm happy to go back and re-adjust if you like the previous one better @ivanblagdan